mysql-test/t/world_database.test

```diff
@@ -0,0 +1,17 @@
+--source include/have_innodb.inc
```

Comment:
While this bug needs a testcase, just trying to load world database is not a very specific one. Is it possible to reduce it?

The testcase should go the InnoDB suite, and probably named percona_ps_260.test or similar

Reply:
During my tests, it was consistently the 181st city that crashed the server, and that's the first table that was created/filled. I could reduce the sql file to include only entries up to that point, but I don't know if that would make it more specific?